Packages/Tracking Preview/HandRays/Runtime/Scripts/InferredBodyPositions/EulerAngleDeadzone.cs

```diff
@@ -94,16 +94,16 @@ public void UpdateDeadzone(float currentAngle)
             float delta = currentAngle + recentredPositionDelta - DeadzoneCentre;
             delta = SimplifyAngle(delta);
 
+            float signDelta = Mathf.Sign(delta);
+            float absDelta = Mathf.Abs(delta);
```

Comment:
Why have these moved before the following if?

They are not used until afterwards (and the if includes a return) so could be unnecessarily being assigned

Reply:
Moved abs delta to reduce duplication - have kept it there and moved sign delta further down the class